### ISSUE TITLE
Build fails on Kotlin/Native

### ISSUE THREAD

**Issue by vyfor** [kordlib/kordx.emoji#25]
Adding `dev.kord.x:emoji:feature-mpp-SNAPSHOT` as a dependency leads to a failure during build, as seen [here](https://github.com/vyfor/kord-test/actions/runs/9990649958) with the following error:

```sh
e: KLIB resolver: Could not find "org.jetbrains.kotlinx:kotlinx-datetime-cinterop-date" in [/home/runner/work/kord-test/kord-test, /home/runner/.konan/klib, /home/runner/.konan/kotlin-native-prebuilt-linux-x86_64-2.0.0/klib/common, /home/runner/.konan/kotlin-native-prebuilt-linux-x86_64-2.0.0/klib/platform/linux_x64]
```

`kordx.emoji` seems to be depending on `kotlinx.datetime` indirectly through `kord-core`. My suspicion is that the latest build (5 months old) of `kordx.emoji:feature-mpp-SNAPSHOT` does not use the latest build (2 weeks old) of `kord-core:feature-native-SNAPSHOT`, thus, causing a [version mismatch](https://github.com/kordlib/kord/commit/3a4c92ae56599fa7d9fac86831e55c5e1ccf0805#diff-697f70cdd88ba88fe77eebda60c7e143f6ad1286bca75017421e93ad84fb87dfR8). To validate this, I built the `feature/mpp` branch locally and published it to Maven local, which resolved the build error. Republishing the snapshot version could potentially solve this issue.

**Comment by DRSchlaubi** [comment]
I'll update the branch, and create a new build this should fix it then, thx for the in-depth report

**Comment by DRSchlaubi** [comment]
Should by fixed by 911fd8f953417732cb312e99a5e2fddd44f5d534
